kolibri/core/content/api.py

```diff
@@ -555,12 +555,12 @@ def consolidate(self, items, queryset):
 
 
 class OptionalPagination(ValuesViewsetCursorPagination):
-    ordering = "id"
+    ordering = "lft"
     page_size_query_param = "max_results"
 
 
 class OptionalContentNodePagination(OptionalPagination):
-    ordering = "id"
```

Comment:
These two lines don't actually need to be here, can just delete them both, as they are inheriting from above.